snow/consensus/snowman/consensus_test.go

```diff
@@ -494,15 +494,15 @@ func RecordPollAcceptSingleBlockTest(t *testing.T, factory Factory) {
 		t.Fatal(err)
 	} else if pref := sm.Preference(); pref != block.ID() {
 		t.Fatalf("Preference returned the wrong block")
-	} else if sm.Finalized() {
+	} else if sm.NumProcessing() == 1 {
```

Comment:
(this comment applies to other places)
```suggestion
	} else if sm.NumProcessing() == 0 {
```

An instance is finalized if there are no processing blocks.